Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+pipeline {
+    agent {
+        dockerfile {
+        }
+    }
+    environment {
+        HOME = .
+    }
+    stages {
+        stage('Setup') {
+            steps {
+                sh 'npm --version'
+                sh 'pwd'
+                sh 'env'
+            }
+        }
+        stage('Build') {
+            steps {
+                sh 'make all'
+            }
+        }
+        stage('Upload - Development') {
+            when {
+                environment name: 'RCH_ENVIRONMENT', value: 'development'
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'jenkins-aws-development', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh 'aws s3 mv ./dist/reach.min.js s3://assets.rch.red/sdk/checkout-web/reach.min.js'
+                }
+            }
+        }
+        stage('Upload - Sandbox') {
+            when {
+                environment name: 'RCH_ENVIRONMENT', value: 'sandbox'
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'jenkins-aws-sandbox', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh 'aws s3 mv ./dist/reach.min.js s3://assets.rch.how/sdk/checkout-web/reach.min.js'
```

Comment:
Could we also make the base URL a jenkins job parameter? or make the base_url keyd off of RCH_ENVIRONMENT?

Reply:
Ive changed it to an environment parameter now.